src/app/modules/main/wallet_section/activity/entry.nim

```diff
@@ -33,6 +33,9 @@ QtObject:
       nftName: string
       nftImageURL: string
 
+      # true for entries that were changed/added in the current session
+      highlight: bool
```

Comment:
This is set for the new elements inserted into the model. It will activate a highlighted background fading for one second as soon as the item comes into the view, see here https://github.com/status-im/status-desktop/blob/d63e561d6c2f5fd4e143dbcf8dc58203bbee6b05/ui/imports/shared/controls/TransactionDelegate.qml#L882